arangod/RestHandler/RestLogHandler.cpp

```diff
@@ -71,6 +73,41 @@ RestStatus RestLogHandler::executeByMethod(
   return RestStatus::DONE;
 }
 
+RestStatus RestLogHandler::handlePutRequest(
```

Comment:
What is the rationale for this change? IMO this is a handler for replication2...

Reply:
Removed it